lib/babe/epoch_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package babe
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
+	"github.com/ChainSafe/gossamer/lib/keystore"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+var keyring, _ = keystore.NewSr25519Keyring() //nolint:typecheck
```

Comment:
when I don't declare keyring here I get an undeclared error, but I then get a lint error since it's defined in `babe_integration_test.go`. how do i fix this?

Reply:
nvm, i just gotta put it in a non integration test file